### PR TITLE
[KO] Add Visual settings, clean up other translations

### PR DESCRIPTION
**Pull request by lechenaultia-korea**

### CODE REVIEW

**Comment by lechenaultia-korea** [comment]
reveiw plz

File: wiki/Visual_Settings/ko.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# 시각적 설정
+
+![Visual Settings Menu](img/visual-settings.jpg "Visual Settings Menu")
+
+**시각적 설정**은 플레이 화면 밑에 가려져 있는 숨겨진 설정입니다.
+게임플레이 도중 일시정지를 했을때나, 비트맵을 불러오고 있을때 변경 가능합니다.
+커서가 설정창 안에서 벗어나기 전까지 비트맵은 일시정지된 상태로 유지됩니다.
```
Comment (cl8n) [inline wiki/Visual_Settings/ko.md:7]: use a single line for paragraphs
```suggestion
**시각적 설정**은 플레이 화면 밑에 가려져 있는 숨겨진 설정입니다. 게임플레이 도중 일시정지를 했을때나, 비트맵을 불러오고 있을때 변경 가능합니다. 커서가 설정창 안에서 벗어나기 전까지 비트맵은 일시정지된 상태로 유지됩니다.
```

File: wiki/Visual_Settings/ko.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# 시각적 설정
+
+![Visual Settings Menu](img/visual-settings.jpg "Visual Settings Menu")
```
Comment (cl8n) [inline wiki/Visual_Settings/ko.md:3]: the alt text (in brackets) and title text (in quotes) should be translated

File: wiki/Visual_Settings/ko.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+**시각적 설정**은 플레이 화면 밑에 가려져 있는 숨겨진 설정입니다.
+게임플레이 도중 일시정지를 했을때나, 비트맵을 불러오고 있을때 변경 가능합니다.
+커서가 설정창 안에서 벗어나기 전까지 비트맵은 일시정지된 상태로 유지됩니다.
+
+*Notice: 시각적 설정을 변경한 것은 osu! 를 끌경우 초기화 됩니다. 모든 비트맵에 설정을 적용하고 싶은 경우나 변경한 설정을 유지하고 싶은 경우에는 [옵션](/wiki/options)항목을 참조해주세요. .*
```
Comment (cl8n) [inline wiki/Visual_Settings/ko.md:9]:
```suggestion
*Notice: 시각적 설정을 변경한 것은 osu! 를 끌경우 초기화 됩니다. 모든 비트맵에 설정을 적용하고 싶은 경우나 변경한 설정을 유지하고 싶은 경우에는 [옵션](/wiki/options)항목을 참조해주세요.*
```

File: wiki/Visual_Settings/ko.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+게임 일시정지 버튼으로 시각적 설정창을 띄울 수 있지만 [멀티플레이](/wiki/Multi)의 경우 방에서 나가지니 주의하세요.
+
+## 설정
+
+| Name | Effect | Notes |
+| :-- | :-- | :-- |
+| `배경의 어두운 정도` | 플레이 화면의 어두운 정도를 조절합니다. (스토리보드와 배경비디오 포함) | 쉬는시간동안 어두운 정도가 최대 30프로까지 밝아집니다. (밝게 하고 싶지 않을경우 option 항목을 이용해주세요) |
+| `스토리보드 끄기` | 스토리보드를 끕니다(kiai time 배경비디오 등등 포함). | 이설정은 간질 위험이 있는 비트맵을 플레이할때 추천됩니다. 만약 비트맵에 스토리보드가 없다면 활성화가 되지 않습니다. |
```
Comment (cl8n) [inline wiki/Visual_Settings/ko.md:18]: missing link to kiai time, check how en.md does it

File: wiki/Visual_Settings/ko.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+| `비트맵 스킨 무시` | 비트맵에 포함되잇는 스킨이 아닌 유저가 선택한 스킨으로 플레이합니다 | 이설정을 바꿀경우 적용되려면 비트맵을 재시작해야 적용됩니다. |
+| `비트맵 히트사운드 무시` | 비트맵이 설정한 히트사운드가아닌 유저가 선택한 스킨에 있는 히트사운드를 사용합니다. | 이설정을 바꿀경우 적용되려면 비트맵을 재시작해야 적용됩니다. |
+| `배경동영상 끄기` | 배경동영상이 실행되지 않습니다. 스토리 보드에잇는 영상이 삭제되는것은 아닙니다. | 이 셋팅은 게임이 시작한이후 활성화 하려면 재시작이 필요합니다. 배경동영상이 없을 경우 활성화 되지 않습니다. |
```
Comment (cl8n) [inline wiki/Visual_Settings/ko.md:21]: i didn't check this so maybe it's already good, but when text is in code formatting (like the "Name"s in this table), it means it comes from the game. so make sure that it matches exactly what the Korean writing says in the game

**Review by cl8n — CHANGES_REQUESTED** [review]
for future commits try to use meaningful names that describe what you're changing, not "Update ko.md" or spam

File: wiki/Visual_Settings/ko.md (hunk 5 above)
Comment (lechenaultia-korea) [reply wiki/Visual_Settings/ko.md:19]: sorry for my rude thing. I have a question "kiai time " is not translated by korean. it means howling in literally but no one think kiai time is howling. so what should i do? 
and other everything is correct korean version writting in the game.

Comment (cl8n) [reply wiki/Visual_Settings/ko.md]: it's fine to leave it in english if the translation would be hard to understand, or if korean players usually call it by the english name

**Review by cl8n — APPROVED** [review]

**Pull request merged**